creusot-contracts/src/std/slice.rs

```diff
@@ -111,8 +113,8 @@ impl<T> SliceIndex<[T]> for usize {
     #[logic]
     #[open]
     #[creusot::why3_attr = "inline:trivial"]
-    fn has_value(self, seq: Seq<T>, out: T) -> bool {
-        pearlite! { seq[self@] == out }
+    fn has_value(self, seq: Seq<T>, out: &T) -> bool {
+        pearlite! { seq[self@] == *out }
```

Comment:
I the end, is this change necessary? If not, then I'd prefer have the one without the reference, it seems cleaner.